src/test/java/uk/co/ramp/people/PopulationGeneratorTest.java

```diff
@@ -85,11 +84,18 @@ public void generate() {
         population.values().stream().map(Case::gender).filter(a -> a.equals(Gender.FEMALE)).count()
             / (double) popSize;
 
+    double hasApp =
+        population.values().stream().map(Case::hasApp).filter(a -> a.equals(true)).count()
```

Comment:
What does `filter(a -> a.equals(true))` do?  (I expect it's useful, but it's not obvious to me!)

Reply:
replaced with ```population.values().stream().filter(Case::hasApp).count()```